index.html

```diff
@@ -424,7 +424,67 @@ <h3>Terminal</h3>
                     </ul>
                 </div>
             </section>
+            <section class="contactme">
+                <div class="contactme__content">
+                    <div class="contactme__details">
+                        <p>
+                            I'm always interested in hearing <br>
+                            about new projects, <br>
+                            so if you'd like to <br>
+                            chat please get in touch.
+                        </p>
+                    </div>
+                    
+                </div>
+                <form class="contactme__content">
+                    <div class="contactme__formgroup">
+                        <input type="text" placeholder="Full Name" class="contactme__input" required>
+                    </div>
+                    <br>
+                    <div class="contactme__formgroup">
+                        <input type="email" placeholder="Email" class="contactme__input" required>
+                    </div>
+                    <br>
+                    <textarea name="contactme" id="" cols="30" rows="10" placeholder="Enter text here" class="contactme__textarea" required></textarea>
+                    <br>
+                    <button class="contactme__submit-button" type="submit">Get In Touch</button>
+                </form>
+            </section>
+
         </main>
+
+        <footer>
+            <div class="footer__container">
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="./images/footer/github.png" alt="no internet">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./images/footer/linkedin_1.png" alt="no internet">
```

Comment:
- [ ] Great job at adding the socials, so everyone can see your media😊, but I see that the margin you put between the socials is not like the Figma, so Kindly try to add the correct margin like the Figma 💯
- [ ] Kindly Also consider putting the correct color on the line which at the top of the socials and raise it little from the socials to be like Figma and to be attractive to the users 😊


> you can see the difference below⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212917699-381411b0-fd15-4dec-bc4a-26cdf777c85e.png) |    ![image](https://user-images.githubusercontent.com/83360636/212917632-0e26e6b4-b06a-45b6-ad2e-b9eebd3fd6b1.png)|